src/test/scala/com/fulcrumgenomics/bam/TrimPrimersTest.scala

```diff
@@ -260,4 +259,55 @@ class TrimPrimersTest extends UnitSpec {
     validate(rec=reads(10), name="q6", firstOfPair=true, fivePrimeSoftClipLength=19) // amplicon 2
     validate(rec=reads(11), name="q6", firstOfPair=false)
   }
+
+  it should "trim fragment reads" in {
+    val amplicons: Seq[Amplicon] = Seq(
```

Comment:
these are the _same_ test cases as the previous test, just now on fragment reads, so the results are verified to be the same :)